druid/src/main/java/org/apache/calcite/adapter/druid/DruidRules.java

```diff
@@ -590,32 +589,21 @@ private static boolean validSortLimit(Sort sort, DruidQuery query) {
         int metricsRefs = 0;
         for (RelFieldCollation col : sort.collation.getFieldCollations()) {
           int idx = col.getFieldIndex();
+          //computes the number of metrics in the sort
           if (idx >= topAgg.getGroupCount()) {
             metricsRefs++;
```

Comment:
_metricsRefs_ is not used anymore (and it seems it is not needed). If it became useless, could we get rid of it?